src/component/Maps/maps.scss

```diff
@@ -0,0 +1,46 @@
+.map-h2 {
```

Comment:
I am not entirely sure where all of this is being rendered. This is why classnames are important!